meta/identical_id.yaml

```diff
@@ -7,10 +7,12 @@ rules:
   patterns:
   - pattern-inside: 'rules: [..., $RULE, ...]'
   - pattern-inside: |
+      ...
```

Comment:
leading ellipsis is an anti-pattern we try to avoid - probably okay here?

cc @minusworld